### PR TITLE
Fixing styling issues in Night Mode w/ new profiles #4361

### LINKED ISSUE #4361: Custom profile text on new Reddit profiles does not show up in night mode.
It looks like this: https://i.imgur.com/G5r40TS.gifv

### PR DESCRIPTION
**Pull request by kevingrahl**
Fixes  #4361

Tested in Chrome and Firefox
I assumed that CSS mixin for the z-index are accepted by this project.

### CODE REVIEW

**Comment by kevingrahl** [comment]
I could have sworn there was a layering issue when I wrote this yesterday with .ProfileTemplate__sidebar, .ProfileBanner and .ProfileSidebar
I just wanted to take screenshots to illustrate why I added those but can't replicate that any more.

File: lib/css/modules/_nightMode.scss [hunk 1]
```diff
@@ -1430,7 +1430,7 @@ $title-link-visited-color: hsl(0, 0%, 65%);
 	.Media.m-profile.m-video {
 		background: $highlight-background-color;
 	}
-	
+
 	.ProfileTemplate__sidebar {
 		background: $highlight-background-color;
 		z-index: 1;
```
Comment (andytuba) [inline lib/css/modules/_nightMode.scss]: I think you accidentally the removing z-indexes.

**Comment by andytuba** [comment]
Layering issue probably got fixed by a deploy on reddit. Now just need to clean up those z-indexes in this PR and it should be  good to go.

**Comment by erikdesjardins** [comment]
Integration test failure is spurious

**Comment by erikdesjardins** [comment]
Thanks!

**Pull request merged**